core/src/main/scala/com.mediative.amadou/SparkJob.scala

```diff
@@ -65,4 +65,23 @@ trait SparkJob extends Logging {
    * measured. For example, many Spark operations are lazy.
    */
   def stage[T](stage: Stage)(f: => T): T = f
+
+  implicit class SparkReadOps(self: DataFrameReader) {
```

Comment:
These implicit conversion classes should go in the `SparkJob` companion object, no? Their types don't need to be tied to specific `SparkJob` instances. Also, let's make them `extends AnyVal` so that they get erased at runtime.
^